omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -284,6 +290,13 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 				us.changeQuestionAnswer(user, secretQuestion, secretAnswer);
 			}
 			
+			//Check if admin wants the person associated with the user to be added to the Provider Table
+			if(addToProviderTableOption != null) {
+				Provider provider = new Provider();
+				provider.setPerson(user.getPerson());
+				provider.setIdentifier(user.getSystemId());
```

Comment:
Was this agreed on, that we should default to systemId as the identifier? We should have tried to emulate the ref app where we prompt the user for an identifier